packages/core/dts.config.js

```diff
@@ -0,0 +1,11 @@
+// Not transpiled with TypeScript or Babel, so use plain Es6/Node.js!
+/**
+ * @type {import('dts-cli').DtsConfig}
+ */
+module.exports = {
+    // This function will run for each entry/format/env combination
+    rollup(config, options) {
+      config.output.name = "JSONSchemaForm";
+      return config; // always return a config.
+    },
+  };
```

Comment:
Adding a new line at the end to avoid the little 
<img width="26" alt="Screen Shot 2023-01-08 at 1 48 55 PM" src="https://user-images.githubusercontent.com/51679588/211220774-2ce67ade-5c82-41a9-a7cf-c055d8856a5f.png">

```suggestion
  };
  
```